services-js/commissions-search/src/Search.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from "react";
+// import './Search.css';
+
+class Search extends Component {
+  _handleKeyPress = event => {
```

Comment:
Can you wrap in a form and bind to `onSubmit` instead?

Reply:
I couldn't get that to work.

Reply:
ok no worries.